packages/circuit-ui/styles/style-mixins.ts

```diff
@@ -195,17 +184,9 @@ export const center = (): SerializedStyles => css`
 /**
  * Visually communicates to the user that an element is focused.
  */
-// TODO: Simplify the function signature in the next major.
-export function focusOutline(
-  options: 'inset',
-): (args?: ThemeArgs) => SerializedStyles;
-export function focusOutline(args?: ThemeArgs): SerializedStyles;
-export function focusOutline(
-  argsOrOptions?: ThemeArgs | 'inset',
-): SerializedStyles | ((args?: ThemeArgs) => SerializedStyles) {
-  if (typeof argsOrOptions === 'string') {
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    return (_args?: ThemeArgs): SerializedStyles => css`
+export function focusOutline(options?: 'inset' | ThemeArgs): SerializedStyles {
```

Comment:
I kept the `ThemeArgs` type here despite it not being used so `focusOutline` can be passed directly to Emotion's `styled` function.